airflow/settings.py

```diff
@@ -465,11 +465,23 @@ def import_local_settings():
     """Import airflow_local_settings.py files to allow overriding any configs in settings.py file."""
     try:
         import airflow_local_settings
-
+    except ModuleNotFoundError as e:
```

Comment:
Looks good - even if there are other import errors laters (it's possible - local imports for example) - then this code was anyhow re-raising them.